HexPos msg
---
bool success
string message